apps/condo/domains/organization/utils/serverSchema/Organization.js

```diff
@@ -10,9 +10,10 @@ const { getById } = require('@core/keystone/schema')
 const { DEFAULT_ROLES } = require('@condo/domains/organization/constants/common')
 const { SBBOL_FINGERPRINT_NAME } = require('@condo/domains/organization/integrations/sbbol/common')
 
-let SALES_CRM_WEBHOOKS_URL = attempt(() => JSON.parse(config.SALES_CRM_WEBHOOKS_URL))
-if (isError(SALES_CRM_WEBHOOKS_URL) || !SALES_CRM_WEBHOOKS_URL.subscriptions || !SALES_CRM_WEBHOOKS_URL.organizations) {
-    SALES_CRM_WEBHOOKS_URL = null
+const logger = pino({ name: 'sales_crm', enabled: falsey(process.env.DISABLE_LOGGING) })
+const SALES_CRM_WEBHOOKS_URL = (config.SALES_CRM_WEBHOOKS_URL) ? JSON.parse(config.SALES_CRM_WEBHOOKS_URL) : null
+if (SALES_CRM_WEBHOOKS_URL && !SALES_CRM_WEBHOOKS_URL.subscriptions && !SALES_CRM_WEBHOOKS_URL.organizations) {
+    throw new Error('Wrong SALES_CRM_WEBHOOKS_URL value')
```

Comment:
won't this fail on build faze ?

Reply:
No failt at build time becuse there is no `SALES_CRM_WEBHOOKS_URL` at that step